javascript/organizing_your_javascript_code/es6_modules.md

```diff
@@ -41,7 +41,7 @@ Read through the npm links below but don't worry about running any of the comman
 
 ### Yarn?
 
-At some point, you will probably run into [Yarn](https://yarnpkg.com/en/) - a replacement for the default `npm`. For the most part, it does the same things, though it _does_ have a few more features. Recent versions of `npm` have incorporated some of the best features of Yarn, so using it won't offer you any real advantages at this point in your career. It _is_ a fine project, however, and may be worth your consideration in the future.
+At some point, you will probably run into [Yarn](https://yarnpkg.com/en/) - a replacement for the default `npm`. For the most part, it does the same things, though it _does_ have a few more features. Recent versions of `npm` have incorporated some of the best features of Yarn, so using it won't offer you any real advantages at this point in your career. It _is_ a fine project manager, however, and may be worth your consideration in the future.
```

Comment:
Replacing "project manager' with "package manager" would be a better way to describe Yarn. What do you think?

Reply:
I agree that's probably a more appropriate way to describe it for this section. I went with 'project manager' because I assumed that's what the original author intended. Trying to fix it now.

Reply:
Everything looks good. Thanks for making the change.